mappings/net/minecraft/entity/passive/FrogBrain.mapping

```diff
@@ -0,0 +1,23 @@
+CLASS net/minecraft/unmapped/C_gtwuaauj net/minecraft/entity/passive/FrogBrain
+	FIELD f_ookmoifl LONG_JUMP_COOLDOWN_RANGE Lnet/minecraft/unmapped/C_hvilmtvi;
+	METHOD m_fhaglmeu addIdleActivities (Lnet/minecraft/unmapped/C_rjqjaxef;)V
+		ARG 0 brain
+	METHOD m_ihwohwid addLongJumpActivities (Lnet/minecraft/unmapped/C_rjqjaxef;)V
```

Comment:
can be shortened to addJumpActivities since frogs kinda just, you know, jump? This can be kept though if it needs to be specified.

Reply:
its specific to long jump tasks